app/components/common/UI/cards/card_info/CardInfo.js

```diff
@@ -2,33 +2,38 @@ export default function CardInfo(props) {
   return (
     <div className="m-auto">
       <div className="p-4">
-        {props.title ? (
+        {props.data?.title ? (
           <h5 className="mb-2 text-base font-bold tracking-tight text-gray-900 dark:text-white">
-            {props.title}
+            {props.data.title}
```

Comment:
i still dont understand how data can be both accessed by an index and an attribute

props.data[0]
props.data.title